docs/partials/install-qemu-sdk.md

```diff
@@ -30,7 +30,7 @@ sudo apt install qemu
 Install the necessary tooling:
 
 ```
-sudo apt install libpcap-dev autoconf automake libtool socat net-tools
+sudo apt install libpcap-dev autoconf automake libtool socat net-tools libglib2.0-dev
```

Comment:
> Added libglib2.0-dev to apt install list to prevent the following error on Ubuntu 20.04: fatal error: glib.h: No such file or directory

During which step does this error pop up?

Could you please add that description to commit body, so `git blame` will actually tell why we recommend installing `libglib2.0-dev` in case somebody will wonder?